build/Makefile

```diff
@@ -447,10 +447,10 @@ setup-prometheus: PVC ?= true
 setup-prometheus: PV_SIZE ?= 64Gi
 setup-prometheus: SCRAPE_INTERVAL=30s
 setup-prometheus:
-	$(DOCKER_RUN) helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+	$(DOCKER_RUN) helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
 	$(DOCKER_RUN) helm repo update
-	$(DOCKER_RUN) helm upgrade prom stable/prometheus \
-		--install --wait --namespace metrics --create-namespace \
+	$(DOCKER_RUN) helm upgrade --install --wait prom prometheus-community/prometheus --version 11.16.2 \
```

Comment:
Was there a reason you moved the `--install` and `--wait` flags before the name of the chart? Now there are some flags before the name and some after, which is odd (same question below as well).

Reply:
No real reason, was just copy pasting the referenced PR. I can switch it around.

Reply:
Fixed!